test/Semantics/allocate11.f90

```diff
@@ -5,19 +5,6 @@
 
 ! Rules I should know when working with coarrays and derived type:
 
-! C736: If EXTENDS appears and the type being defined has a coarray ultimate
-! component, its parent type shall have a coarray ultimate component.
```

Comment:
C736 is specific to the definition of a component of a derived type, and all of the tests in allocate11.f90 were allocate statements.  I added the test resolve86.f90 in my previous pull request to cover the case of defining a component that contains a coarray ultimate component without having a parent type that contains a coarray ultimate component.  This may have happened after you reviewed and approved that pull request.